### DISCUSSION TITLE
Partial of vector V with respect the normal vector

### DISCUSSION THREAD

**Discussion by TLWise** [idaholab/moose#26844] [Q&A Modules: General] [answered]
Is there an example code that implements the partial derivative of vector V with respect the partial derivative of the normal vector, with that operation then multiplied by the normal direction, example n_y?

**Comment by lindsayad** [comment]
This is different than the projection of the gradient onto the normal, e.g. $(\nabla \vec{u} \cdot \vec{n})\vec{n}$ ?

**Reply by TLWise** [reply]
Is there an example post processor that incorporates this?

**Reply by GiudGiud** [reply, marked as answer]
I dont think so. We have (grad_u * n) in SideDiffusiveFluxIntegral but no multipliciation by the normal after.
It still shows how to get the gradient and what the normal vector is in the code

It should be pretty easy to make a new postprocessor though. 
Once you know grad_V, computing dV/dn is simple math, and then multiplying by n_y is trivial too

**Reply by TLWise** [reply]
Thanks for reassuring me that there is a means to solve the formulation of this type of problem in the software. I have some more homework to do on the subject.

**Reply by TLWise** [reply]
IntegralDirectedSurfaceForce is exactly what I'm attempting to accomplish, but there is no finite element implementation of it.
